packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/canHoldWith..st

```diff
@@ -0,0 +1,4 @@
+accessing
+canHoldWith: aNumber
```

Comment:
Maybe call this canReceive or add a comment, it is not like the node does not hold (as in collapses) with the input.